test/cpp/jit/tests.h

```diff
@@ -130,12 +130,13 @@ namespace jit {
   _(GPU_FusionBinaryOps)         \
   _(GPU_FusionTernaryOps)        \
   _(GPU_FusionCompoundOps)       \
-  _(GPU_FusionCastOps)           \
```

Comment:
This may not be part of the Merge as Naoya fixed this yesterday, but I will point out FusionCastOps test being removed.

Reply:
Yes, it was added with #49

Reply:
Added back in.